settings/chains/prod/sonic.yaml

```diff
@@ -0,0 +1,18 @@
+network_name: sonic
+chain_id: 146
+rollup_type: _
+dao:
+  crv: ""
+  emergency_admin: ""
+  ownership_admin: ""
+  parameter_admin: ""
+  vault: ""
+explorer_base_url: https://sonicscan.org/
+# Not related to development, for further integrations
+layer: 1
+logo_url: https://assets.coingecko.com/coins/images/38108/standard/Logomark_Sonic_Black.png?1734534767
+native_currency_symbol: S
```

Comment:
could you please add additional information here. see example: https://github.com/curvefi/curve-core/blob/main/settings/chains/devnet/hyperliquid.yaml